test/unique.test.ts

```diff
@@ -69,8 +69,8 @@ describe("Unique", function () {
     })(
       {
         plugins: [
-          new webpack.HotModuleReplacementPlugin(),
           new webpack.DefinePlugin({}),
+          new webpack.HotModuleReplacementPlugin(),
```

Comment:
Can you duplicate the original test and do your change so we are sure there's no regression? I think after that this one is good to go. Thanks a lot!

Reply:
Done. I have also added a note to the documentation on the ordering of elements after a merge.

Reply:
Awesome. I'll review, merge, and publish sometime today. 👍